docs/sources/next/using-k6/modules.md

```diff
@@ -355,10 +355,7 @@ $ k6 run dist/signup.bundle.js \
 
 ## Use TypeScript
 
-k6 does not natively support TypeScript. If you wish to write k6 tests in Typescript, you will need a bundler, as demonstrated in the previous examples:
-
-- Using Webpack: Refer to [k6-template-typescript](https://github.com/grafana/k6-template-typescript) and [k6-jslib-aws](https://github.com/grafana/k6-jslib-aws).
-- Using Rollup: Apply the [@rollup/plugin-typescript](https://github.com/rollup/plugins/tree/master/packages/typescript) to the [k6-rollup-example](https://github.com/grafana/k6-rollup-example).
+k6 supports partial TypeScript support with the `experimental_enhanced` compatibility mode. For more details, refer to [JavaScript and TypeScript compatibility mode](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/javascript-typescript-compatibility-mode/).
```

Comment:
@joanlopez @szkiba we also have this other mention of TypeScript, do we still wanna mention the Webpack and Rollup examples to users, or should we just link to the new compatibility mode?

Are there still pros to using the Webpack and Rollup examples over the new mode?

Reply:
@szkiba is probably a better candidate to answer here

Reply:
I would prefer the new compatibility mode (experimental_enhanced) instead of bundlers (webpack, rollup), that's why it was made. Maybe I would add a link to @ppcano 's blog, it summarizes the topic well.

Reply:
Agree with Ivan. I think we should not reference these Bundlers for TypeScript anymore. 

We already mention the template Bundlers for bundling npm modules and on the Read more section.

Reply:
That's a good point that we already mention the templates at the end of the page. :)

I guess we might want to update your [blog post](https://grafana.com/blog/2024/05/02/setting-up-your-grafana-k6-performance-testing-suite-javascript-tools-shared-libraries-and-more/) as well Pepe to include this new mode in case people run into it. Would you mind reaching out to the content team to do that?

Reply:
I'll mark this as a resolved since this is separate from this PR. 🤓